run_onnx.py

```diff
@@ -22,25 +46,53 @@ def run_onnx_model() -> None:
         "buffer.1": np.zeros(574).astype(np.float32),
     }
 
+    def get_angular_velocity() -> np.ndarray:
+        return imu.gyr_rate()
+
+    def get_euler_angles() -> np.ndarray:
+        return kalman_filter.step()
+
     def get_joint_angles() -> np.ndarray:
-        return np.zeros(10).astype(np.float32)
+        angles = np.zeros(10).astype(np.float32)
```

Comment:
this should be explicitly called using enums or dataclass for the robot joints definition